import.py

```diff
@@ -245,7 +245,15 @@ def update_content(base, target, git_repo, content_dir, msg, user):
     LOGGER.info("Checking out target_branch: %s", target)
     git_repo.git.checkout('-B', target)
     git_repo.git.rm('-rf', '*')
-    copy_tree(content_dir, git_repo.working_dir)
+    try:
+        shutil.copytree(
```

Comment:
Why the shift from distutil to shutil? Does it do the read permissions ignoring that you want it to?

Reply:
It will not throw immediately on permission errors, it will collect all then throw hence I can just try catch and log the warning instead.